Android/README.md

```diff
@@ -1,13 +1,9 @@
 # Python for Android
 
 These instructions are only needed if you're planning to compile Python for
-Android yourself. Most users should *not* need to do this. If you're looking to
-use Python on Android, one of the following tools will provide a much more
-approachable user experience:
-
-* [Briefcase](https://briefcase.readthedocs.io), from the BeeWare project
-* [Buildozer](https://buildozer.readthedocs.io), from the Kivy project
-* [Chaquopy](https://chaquo.com/chaquopy/)
+Android yourself. Most users should *not* need to do this. Instead, use one of
+the tools listed in Doc/using/android.rst, which will provide a much easier
```

Comment:
```suggestion
the tools listed in [`Doc/using/android.rst`](https://docs.python.org/3/using/android.html), which will provide a much easier
```

Reply:
I think it's better for references to the Doc directory not to have URLs, because they would take the reader to the stable version of the docs, not the version that corresponds to their current branch.

Reply:
Fair enough:

```suggestion
the tools listed in `Doc/using/android.rst`, which will provide a much easier
```